src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateVersion = [];
```

Comment:
There are multiple versions stored, right?
```suggestion
  const stateVersions = [];
```